integration-tests/features/check-target.feature

```diff
@@ -14,3 +14,18 @@ Scenario: Checking for already used names on target VM
          | storage2   | macrocontainer dir  |
     Then checking target usability should take less than 10 seconds
      And all claimed names should be reported exactly once
+
+Scenario: Checking for available services on target VM
+   Given the local virtual machines:
+         | name       | definition          | ensure_fresh |
+         | target     | centos7-target      | no           |
+     And the following names claimed on target:
+         | name       | kind                |
+         | app1       | idle macrocontainer |
+         | app2       | idle macrocontainer |
+         | container1 | idle container      |
+         | container2 | idle container      |
+         | storage1   | macrocontainer dir  |
+         | storage2   | macrocontainer dir  |
+    Then checking target services status should take less than 10 seconds
+     And all status reported are correct
```

Comment:
This might be clearer for readers if we spelled out the expectations as distinct steps:

```
 And docker should be reported as "ok"
 And rsync should be reported as "ok"
 And all claimed names should be reported exactly once
```

To allow the existing definition for the last step to be re-used, `run_check_target_status` could do `context._reported_names = context._target_status['containers']`